### DISCUSSION TITLE
Element/Matrix channel offline?

### DISCUSSION THREAD

**Discussion by peter-scholtens** [woodpecker-ci/woodpecker#3772] [Ask a Question]
The [homepage](https://woodpecker-ci.org/) advertises the use of a Matrix channel **#woodpecker:matrix.org**. Is that still active? Or is it moved somewhere else? Would be nice to adapt the homepage to the actual situation.

**Comment by qwerty287** [comment]
The link works for me. Can you check again?

**Comment by peter-scholtens** [comment]
Via PC (Linux Mint with Element-app) it failed. But with mobile phone (Android 13) it now somehow worked.

1. But it is advertised as a single channel, while I see multiple ones...?

2. And it is not searchable in the public room list of the server matrix.org? (I tried that, as the first link method failed for me)

**Reply by qwerty287** [reply]
> But it is advertised as a single channel, while I see multiple ones...?

Where is it advertised as single channel? It's a space, but only referred as "Matrix" I think. The address of the "general" room is #woodpecker-ci:matrix.org if you still need it

**Reply by peter-scholtens** [reply]
Weirdly enough it seems now Firefox (on Linux Mint) has learned to remember the correct location and starts element-desktop with the correct link (using element://vector...), even after logging out (space and rooms) several times. Earlier it forwarded me to an ElementWeb instance located at https://app.element.io.
Problem (1) is solved, I only wonder why is not visible (2) in the public room/spaces list of matrix.org? Would be handy, for others who failed to login the direct link method?

**Reply by qwerty287** [reply]
> Would be handy, for others who failed to login the direct link method?

That's right. I checked it, but i didn't find an option how to publish. Either I'm not allowed to do so or this is more complicated. Maybe @woodpecker-ci/owner can check this?

**Reply by anbraten** [reply]
Tried, but it is somehow throwing a permission error for me.

**Comment by qwerty287** [comment]
closing this as was found
